### DISCUSSION TITLE
expected stored sv record after soft delete Strategies and validator

### DISCUSSION THREAD

**Discussion by elmalakomar** [mia-platform/community#441] [Q&A] [answered]
### What product are you having troubles with?

Fast Data

### What Console version are you using?

v12.0.2

### Description

In case the single views generated doesn't satisfy the validator function, with a soft delete I am wrong to expect that a could have the newly generated single view but in state "DELETED"?
Right now I get the old version of the single view with the change of state.

### Actual Outcome

old single view record with change of state to DELETED

### Expected Outcome

updated single view record in state DELETED

**Comment by danibix95** [comment, marked as answer]
Hi @elmalakomar, currently the Fast Data configurator allows to define your own custom [Upsert and Delete strategies](https://docs.mia-platform.eu/docs/fast_data/configuration/single_view_creator/plugin#upsert-and-delete-strategies).

Considering your use case, I would rather specify a custom logic that addresses and solves it, instead of changing the current underlying logic of the service.
